CMS-Enterprise/mint-app
# [MINT-3106] Pre-launch Cleanup

# MINT-3106

## Description

### Acceptance Criteria

- Models you're following table will need to update FFS CRs and TDLs
- Models with FFS CRs and TDLs will need to be updated
- Remove the ability to add a CR/TDL in MINT
- Remove FFS CR/TDL section from the Model Plan task list
- Remove FFS CR/TDLs from the CSV file

### Additional Details

- [Figma](https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=2852-95648&t=NOqkTpmuDmBfY6dk-4): Adjustments to existing task list
- [Figma](https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=2852-97442&t=NOqkTpmuDmBfY6dk-4): Clean up to existing tables

## How to test this change

1.  Verify that you cannot add a new CR and/or TDL
2. Task list to match figma
3. Favorite Model Plans have the correct CR/TDL shown
4. Models with FFS CRs and TDLs table have correct CR/TDL shown
5. correct CR/TDL shown in read only view

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
